pkg/compute/compute_integration_test.go

```diff
@@ -51,7 +51,7 @@ func TestInit(t *testing.T) {
 		},
 		{
 			name:       "unkown repository",
-			args:       []string{"compute", "init", "--from", "https://example.com/template"},
+			args:       []string{"compute", "init", "--from", "https://example.com/template", "--backend-port", "80"},
```

Comment:
@phamann I'm having to pass this flag in everywhere to prevent the integration tests from failing. Which is a smell.

The fundamental issue is that there is no 'prompt' in the integration test and so if I don't pass the flag, then the code will error when it tries to parse an empty string into an integer (because the code logic expects a user to have entered some value when prompted).

I'd ideally like to mock the behaviour somehow so that I don't need to pass the flag everywhere, but I'm not sure of the best approach to achieve that 🤔

Reply:
One thing I could do, without mocking (and would mean I could remove the `--backend-port` flag from every test setup), is if we get an error when trying to parse the given port number from the user, to just default to port 80 rather than return the error (we would then have to display a message to the user so they were aware that there was an issue).

In reality, the only time we'd get an error parsing is if someone purposely provides an invalid value (like "xyz") or if they make a mistake like prefixing the port number with a colon ":443" otherwise I don't expect users to see this sort of error.